modules/ipi-install-preparing-to-deploy-with-virtual-media-on-the-baremetal-network.adoc

```diff
@@ -0,0 +1,119 @@
+// This is included in the following assemblies:
+//
+// installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+[id="preparing-to-deploy-with-virtual-media-on-the-baremetal-network_{context}"]
+= Preparing to deploy with Virtual Media on the baremetal network
+
+If the `provisioning` network is enabled and you want to expand the cluster using Virtual Media on the `baremetal` network, use the following procedure.
+
+.Procedure
+
+. Edit the `provisioning` custom resource (CR) to enable deploying with Virtual Media on the `baremetal` network:
```

Comment:
```suggestion
. Edit the `provisioning` custom resource (CR) to enable deploying with Virtual Media on the baremetal network:
```

Removing the backticks from bare metal as it is not used anywhere else in the assembly.